### PR TITLE
add versions check

### PR DESCRIPTION
**Pull request by StevenTobin**
Add a check to ensure we don't merge a CHANGEME in the types file

### CODE REVIEW

File: ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master.yaml [hunk 1]
```diff
@@ -69,6 +69,10 @@ tests:
   commands: make manifest/check
   container:
     from: src
+- as: version
```
Comment (mikenairn) [inline ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master.yaml:72]: Might be better called versions?

File: ci-operator/jobs/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master-presubmits.yaml [hunk 2]
```diff
@@ -349,3 +349,48 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )vendor,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (mikenairn) [inline ci-operator/jobs/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master-presubmits.yaml:353]: We could make this optionally run only when that file changes?

File: ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master.yaml (hunk 1 above)
Comment (StevenTobin) [reply ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master.yaml:72]: Sure, I thought it was versions already, must have mistyped

File: ci-operator/jobs/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master-presubmits.yaml (hunk 2 above)
Comment (StevenTobin) [reply ci-operator/jobs/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master-presubmits.yaml:353]: Yeah great, I actually missed that I could do that, i'll make the change

Comment (mikenairn) [reply ci-operator/jobs/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master-presubmits.yaml:353]: `pkg/apis/integreatly/v1alpha1/rhmi_types.go`

Comment (mikenairn) [reply ci-operator/jobs/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master-presubmits.yaml:353]: see https://github.com/openshift/release/blob/master/ci-operator/jobs/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master-presubmits.yaml#L17

Comment (mikenairn) [reply ci-operator/jobs/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master-presubmits.yaml:353]: https://github.com/openshift/release/blob/master/ci-operator/jobs/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master-presubmits.yaml#L4

**Comment by mikenairn** [comment]
This looks fine to me, but we will need to wait until this is merged back to master https://github.com/integr8ly/integreatly-operator/pull/1087

/hold

**Comment by mikenairn** [comment]
I'm not sure what we are going to do with this longer term, but there will be no harm in having this in place anyway.

/lgtm
/approve

**Pull request merged**